### PR TITLE
Added Initializing auth client from config method

### PR DESCRIPTION
**Pull request by pmahindrakar-oss**
Signed-off-by: Prafulla Mahindrakar <prafulla.mahindrakar@gmail.com>

# TL;DR
_Please replace this text with a description of what this PR accomplishes._

## Type
 - [ ] Bug Fix
 - [ ] Feature
 - [ ] Plugin

## Are all requirements met?

 - [ ] Code completed
 - [ ] Smoke tested
 - [ ] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
 _How did you fix the bug, make the feature etc. Link to any design docs etc_

## Tracking Issue
https://github.com/lyft/flyte/issues/<number>

## Follow-up issue
_NA_
OR
_https://github.com/lyft/flyte/issues/<number>_

### CODE REVIEW

File: clients/go/admin/client.go [hunk 1]
```diff
@@ -144,6 +150,18 @@ func InitializeAdminClient(ctx context.Context, cfg Config) service.AdminService
 	return NewAdminClient(ctx, adminConnection)
 }
 
+// Create an AdminClient and AuthServiceClient with a shared Admin connection for the process
+func InitializeClients(ctx context.Context, cfg Config) (service.AdminServiceClient, service.AuthServiceClient) {
```
Comment (EngHabu) [inline clients/go/admin/client.go:154]: I think we might want to create and expose a wrapper similar to how kube api client behaves... so you get back one client that has .Admin() and .Auth()

Comment (pmahindrakar-oss) [reply clients/go/admin/client.go]: Done

File: clients/go/admin/client.go [hunk 2]
```diff
@@ -26,11 +27,32 @@ var (
...
 )
 
+// Clientset contains the clients exposed to communicate with various admin services.
+type Clientset struct {
+	AdminServiceClient service.AdminServiceClient
+	AuthServiceClient  service.AuthServiceClient
+}
+
+// AdminClient retrieves the AdminServiceClient
+func (c *Clientset) AdminClient() service.AdminServiceClient {
```
Comment (EngHabu) [inline clients/go/admin/client.go:37]:
```suggestion
func (c Clientset) AdminClient() service.AdminServiceClient {
```

File: clients/go/admin/client.go [hunk 3]
```diff
@@ -26,11 +27,32 @@ var (
...
+	return c.AdminServiceClient
+}
+
+// AuthClient retrieves the AuthServiceClient
+func (c *Clientset) AuthClient() service.AuthServiceClient {
```
Comment (EngHabu) [inline clients/go/admin/client.go:42]:
```suggestion
func (c Clientset) AuthClient() service.AuthServiceClient {
```

**Review by EngHabu — APPROVED** [review]

File: clients/go/admin/client.go (hunk 2 above)
Comment (pmahindrakar-oss) [reply clients/go/admin/client.go]: Done

File: clients/go/admin/client.go (hunk 3 above)
Comment (pmahindrakar-oss) [reply clients/go/admin/client.go]: Done

**Pull request merged**